<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="choicesLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choices</string>
    </property>
   </widget>
   <widget class="QListWidget" name="choicesList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>841</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="optionLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Option:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="optionInput">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decideButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>MAKE A CHOICE!</string>
    </property>
   </widget>
   <widget class="QLabel" name="weightLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="weightSpin">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
